htlcswitch/link.go

```diff
@@ -879,6 +879,13 @@ func (l *channelLink) htlcManager() {
 			// what they sent us before.
 			// TODO(halseth): ban peer?
 			case err == lnwallet.ErrInvalidLocalUnrevokedCommitPoint:
+				err = l.channel.MarkBorked()
+				if err != nil {
+					log.Errorf("Unable to mark channel "+
+						"borked: %v", err)
+					return
```

Comment:
Why do we return here? Shouldn't we proceed to fail the link?

Reply:
Yep, done!